docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+
+This document outlines the rules and guidelines that apply to the OBELISK project. All contributors are expected to strictly follow them to ensure that the workflow is consistent and the repository is easy to maintain. Feel free to ask any questions regarding contributions or any other topic.
+
+Please note that this document might (and probably will) change in the future. By contributing you are expected to keep up with all of the updates.
+
+
+## Branches
+
+There are three types of branches:
+- `release branch` - which can be created and merged to from `develop` by DevOps only. For example `release-v1.2.1`
+- `develop branch` - we merge feature branches to this branch
+- `feature branch` - a branch that introduces specific feature. It must be named consistently with Jira issue ID - for example `OK-215`
+
+Branches that don't fall under any of the branch types listed above are not allowed.
+
+Pushing with `--force` is not allowed. Pushing with `--force-with-lease` is allowed only on the feature branches and under the condition that you work on the branch alone, which will be the case for most branches.
+
+
+## Commits
+
+We use slightly modified version of [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). A special hook has been prepared for this purpose. It can be found [here](https://github.com/OBELISK-TEAM/OBELISK/tree/develop/.githooks/prepare-commit-msg). In order to use it, copy the `prepare-commit-msg` file to `.git/hooks` directory. Using it on Linux might also require changing the execution privelleges: `chmod +x prepare-commit-msg`. To run the hook, use the following command: `git commit -a`.
```

Comment:
A small thing to correct
```suggestion
We use slightly modified version of [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). A special hook has been prepared for this purpose. It can be found [here](https://github.com/OBELISK-TEAM/OBELISK/tree/develop/.githooks/prepare-commit-msg). In order to use it, copy the `prepare-commit-msg` file to `.git/hooks` directory. Using it on Linux might also require changing the execution permissions: `chmod +x prepare-commit-msg`. To run the hook, use the following command: `git commit -a`.
```